persistence/src/main/java/com/epam/esm/persistence/util/EncryptDecriptUtil.java

```diff
@@ -0,0 +1,109 @@
+package com.epam.esm.persistence.util;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.spec.DESKeySpec;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+
+/**
+ * Sensitive data encryptor-decryptor.
+ *
+ * @author Yury Zmushko
+ * @version 1.0
+ */
+public class EncryptDecriptUtil {
```

Comment:
I think that will be better to separate this class to several - EncryptData and DecryptData

Reply:
I'd prefer keep it in one class. Anyway, i deleted it